anitya/ui.py

```diff
@@ -567,6 +566,44 @@ def new_project():
         400,
     )
 
+from flask import Flask, request, jsonify
+@ui_blueprint.route('/get_default_values_for_backend', methods=['GET'])
+def get_default_values_for_backend():
```

Comment:
This isn't what I wanted in this feature.
It should be a new API endpoint, which will scan the project on the server and fills in the values obtained from the backend.

Like for example, user adds URL `https://pypi.org/project/<name>` and you fill the new_project page with all the data obtained by the API call.

To see what I have in mind, look at the https://www.protondb.com/contribute and how they can find the game based on the AppID, URL etc.

Reply:
OKkay if I get You well, then the user should enter the url, then the API is called and retrieves data related to that url entered, 
And which field is responsible for the url here

I will try that out
![image](https://github.com/fedora-infra/anitya/assets/121938133/1a723d10-4c79-4817-9f28-78e901a65e20)

Reply:
could this be a sample url 
https://release-monitoring.org/project/250162/

Reply:
I've read through the docs and I was looking for the API endpoint that could query through the projects and return their data maybe as JSON, but happens not to be there.
Any idea please?